src/Content/Feature.php

```diff
@@ -23,13 +23,15 @@ public static function isEnabled($uid, $feature)
 	{
 		$x = Config::get('feature_lock', $feature, false);
 
-		if ($x === false) {
+		if (is_null($x)) {
```

Comment:
Please don't change anything in Feature.php. This file works, I fixed it at the hackathon. For your information: Here we are using the default value (third parameter) which we set to "false". So a "is_null" would be wrong.

Reply:
ahh. ok I see that now. my mistake